modules/test/playwright/pages/object-web/ModelBuilderPage.ts

```diff
@@ -48,6 +51,9 @@ export class ModelBuilderPage {
 		this.leftSidebarItems = page.locator(
 			'li.treeview-item div.autofit-col'
 		);
+		this.modalDeleteButton = page
+			.locator('button.btn-danger')
+			.filter({hasText: 'Delete'});
```

Comment:
Hey @rafaellajordao, remember the [playwright locators quick quide](https://playwright.dev/docs/locators). 

For this case you could use `getByTitle('Delete')`, but notice that is the same locator the `deleteObjectRelationshipButton`. So you can rename the locator `deleteObjectRelationshipButton` to more generic name and use it for your case.

Reply:
I fixed it! :smile_cat: